scripts/build-schema.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+set -e
+set -o pipefail
+
```

Comment:
This so the script fails if a subprocess fails - the pre-application schema was failing but it continued on and there was just a single 'Error' line so it was hard to spot in the output.